domains/dylan.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "This will soon be a dev portfolio for myself :)",
+  "repo": "https://github.com/dylanjamesdev",
+  "owner": {
+    "username": "dylanjamesdev",
+    "email": "dylan@tritan.gg"
+  },
+   "record": {
+    "URL": "https://gitlab.com/team-tritan
```

Comment:
Hey, here you need to replace the record with: `"URL": "https://gitlab.com/team-tritan"`, the ending quotation marks are missing.

Reply:
Ahh my bad

Reply:
Fixed